pkg/update/update.go

```diff
@@ -101,3 +101,29 @@ func TrimVersionInfo(pkg string) (string, string) {
 
 	return name, version
 }
+
+// This function compares the devVersions and the runtimeVersions
+func CompareVersions(a, b []string) bool {
```

Comment:
Function should be renamed to `ComparePackages`. 
Also, the comment should start with `ComparePackages ` so linter stays happy as a Go convention